VERSION

```diff
@@ -1 +1 @@
-1!4.1.1
+1!4.1.2
```

Comment:
This library follows semantic versioning, it feels like we are adding feature and we should bump the minor part:

```suggestion
1!4.2.1
```